src/Compilers/CSharp/Test/Symbol/Symbols/ExtensionMethodTests.cs

```diff
@@ -3615,5 +3615,54 @@ public static void Foo(this int x)
                 //     using X = N.S;
                 Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using X = N.S;").WithLocation(4, 5));
         }
+
+        [WorkItem(1094849, "DevDiv")]
```

Comment:
Please port issues to github when possible (scrubbing private data if necessary) so others can make as much sense of pull requests as we can.